### PR TITLE
Fix spelling of bootstrap and add hot reloading on white/blacklist on bootstrap

### PR DESCRIPTION
**Pull request by AurelienFT**

### CODE REVIEW

**Comment by aoudiamoncef** [comment]
OMG, I thought that iI fixed it in another PR but Nope I didn't.Sorry

File: massa-bootstrap/src/server.rs [hunk 1]
```diff
@@ -139,26 +106,63 @@ struct BootstrapServer {
...
-        //     BootstrapPeers,
-        //     Arc<RwLock<FinalState>>,
-        // )> = None;
-        // let cache_timer = sleep(cache_timeout);
+        let cache_timeout = self.bootstrap_config.cache_duration.to_duration();
+        let (mut whitelist, mut blacklist) = reload_whitelist_blacklist(
+            &self.bootstrap_config.bootstrap_whitelist_path,
+            &self.bootstrap_config.bootstrap_blacklist_path,
+        )?;
+        let cache_timer = sleep(cache_timeout);
```
Comment (aoudiamoncef) [inline massa-bootstrap/src/server.rs:163]: I imagine that we would refresh the cache every X minutes, why not use [interval](https://docs.rs/tokio/latest/tokio/time/fn.interval.html)

Comment (AurelienFT) [reply massa-bootstrap/src/server.rs]: Done.

**Comment by AurelienFT** [comment]
bors merge

**Pull request merged**